File: Algoritmos Traduzidos/Perl/Algoritmos_Luis/59.pl
#Entrar com os valores dos catetos de um triângulo retângulo e imprimir a hipotenusa.

print "introduza o 1o cateto\n";
$cateto1 = <STDIN>;
print "introduza o 2o cateto\n";
$cateto2 = <STDIN>;
$h = sqrt(($cateto1**2)+($cateto2**2));
print "a hipotenusa e $h\n";

File: Algoritmos Traduzidos/Ruby/Algoritmos_Luis/59.rb
#Entrar com os valores dos catetos de um triângulo retângulo e imprimir a hipotenusa.
puts "introduza o 1o cateto"
cateto1 = gets
puts "introduza o 2o cateto"
cateto2 = gets
h = Math.sqrt((cateto1.to_f**2)+(cateto2.to_f**2))
puts "a hipotenusa e #{h}"